docs/types.rst

```diff
@@ -96,7 +96,19 @@ In Solidity, division rounds towards zero. This mean that ``int256(-5) / int256(
 Note that in contrast, division on :ref:`literals<rational_literals>` results in fractional values
 of arbitrary precision.
 
-Division by zero and modulus with zero throws a runtime exception.
+The modulo operator ``%`` yields the remainder ``r`` after the division of the left operand ``a``
+and the right operand ``n``, where ``q = int(a / n)`` and ``r = a - (n * q)``. Note that
+``a % n == -(abs(a) % n)`` holds for negative ``a``.
+
+This means that modulo results in the same sign as its left operand (or zero):
```

Comment:
Perhaps rather use this as the definiton and then mention the above as a consequence?